src/exportJarSteps/utility.ts

```diff
@@ -81,3 +90,21 @@ export function successMessage(outputFileName: string) {
             }
         });
 }
+
+export function toPosixPath(inputPath: string): string {
+    return inputPath.split(win32.sep).join(posix.sep);
+}
+
+export async function getExtensionApi(): Promise<any> {
+    const extension: Extension<any> | undefined = extensions.getExtension("redhat.java");
+    if (extension === undefined) {
+        failMessage("Language Support for Java(TM) by Red Hat isn't running, the export process will be aborted.");
+        return Promise.reject();
```

Comment:
Because the caller of this method returns a Promise. The try-catch block which to hand errors is in the caller of this caller. If we use `throw`, it may not be caught by the catch statement.
For this function, I will update a commit, which uses `reject` with error messages and catches them outside instead of using `failMessage` directly.